builder/tests/builder/src/main/java/io/helidon/builder/test/testsubjects/CustomNamedBlueprint.java

```diff
@@ -16,58 +16,47 @@
 
 package io.helidon.builder.test.testsubjects;
 
-import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.TreeMap;
-import java.util.TreeSet;
 
-import io.helidon.builder.Annotated;
-import io.helidon.builder.Builder;
-import io.helidon.builder.Singular;
+import io.helidon.builder.api.Prototype;
 
 /**
  * Used for demonstrating and testing the Builder.
  * <p>
  * In this case we are overriding the Map, Set, and List types, usages of annotations placed on the generated class, as well as
  * changing the package name targeted for the generated class.
  */
-@Builder(packageName = ".impl",
-         implPrefix = "Default",
-         implSuffix = "",
-         listImplType = LinkedList.class,
-         mapImplType = TreeMap.class,
-         setImplType = TreeSet.class,
-         requireBeanStyle = true)
-@Annotated("com.fasterxml.jackson.annotation.JsonAutoDetect(fieldVisibility = "
+@Prototype.Blueprint(beanStyle = true)
```

Comment:
I do think we will need an issue enhancement to support TreeMap, TreeSet, etc. These are very useful and we shouldn't lose that capability.

Reply:
We are not building a generic solution to handle everything. We do not need it in Helidon. We may add it in the future if the use case is compelling. This should be an MVP

Reply:
If we need this in Helidon, we will fix it